main.go

```diff
@@ -139,6 +147,35 @@ func runCommand(command string, args []string) (string, error) {
 	return outb.String(), err
 }
 
+func retryCommand(command string, args []string) (out string, err error) {
+	for i := 1; i <= 3; i++ {
+		out, err = runCommand(command, args)
+		if out == "" && err != nil {
+			time.After(jitter(i) + 1*time.Microsecond)
+		} else {
+			return
+		}
+	}
+	return
+}
+
+func jitter(i int) time.Duration {
```

Comment:
This seems quite long compared to https://github.com/estafette/estafette-foundation/blob/master/foundation.go#L95-L101

Reply:
ok, I'll check that out.

I got this logic from pester: https://github.com/sethgrid/pester/blob/master/pester.go#L136

Reply:
also this will be in milliseconds, are you sure it's too long?

Reply:
ok, I removed the exponential part of the jitter, it was really too long
I think we can keep it linear

Reply:
I meant long in the number of lines of code, not the duration :) For any action that no real person is awaiting it's best to use exponential backoff; for any customer facing request you would use linear backoff. So I think it's better to use exponential backoff as you did before. I guess copying this from pester is fine, that should be pretty much battle hardened. I don't think you want to retry after milliseconds though, so perhaps return seconds instead? Keeping them somewhere between 10-30 seconds for the max number of retries? That should be acceptable for a build, rather than failing it.

Reply:
I reverted to exponential.
But if I change it to seconds the implementation is going really different from pester.
For the max retries (3) will be something around 9000ms, so ~9 seconds. I think this is ok.